search/api.py

```diff
@@ -228,6 +235,27 @@ def create_user_mapping():
             'state_or_territory': NOT_ANALYZED_STRING_TYPE,
         }},
     })
+    mapping.field('enrollments', 'nested', properties={
+        'course_details': {
```

Comment:
just wondering: what do we gain from defining these specific properties in this mapping (as opposed to just defining it as `mapping.field('enrollments', 'nested')`)